app/components/article-featured-video.js

```diff
@@ -24,6 +24,46 @@ export default Component.extend(
 			this._super(...arguments);
 
 			this.initVideoPlayer();
+			this.initOnScrollBehaviour();
+		},
+
+		/**
+		 * Manages video transformation on user's scroll action
+		 *
+		 * @returns {void}
+		 */
+		initOnScrollBehaviour() {
+			let $video = this.$('.video-container'),
+				$siteHead = $('.site-head'),
+				videoBottomPosition = $video.offset().top + $video.height(),
+				showVideoOnScroll = true;
+
+			$(window).on('scroll', {player: this.player, $window: this.$(window)}, function ({data: {player, $window}}) {
+				run.throttle(this, function () {
+					let currentScroll = $window.scrollTop();
+
+					if (currentScroll >= videoBottomPosition && !$video.hasClass('fixed')) {
+						if (showVideoOnScroll && (player === undefined || ! player.isPlaying())) {
+							$video.addClass('fixed');
+							$siteHead.addClass('no-shadow');
```

Comment:
BTW `site-head` isn't a part of this component. Modifying it's attributes breaks encapsulation. To achieve the goal you should send an action up to the first common parent and then pass data to `site-head`. It's called Data Down Action Up approach in Ember: https://dockyard.com/blog/2015/10/14/best-practices-data-down-actions-up